src/helpers/clipboard/clipboard_cells_state.ts

```diff
@@ -479,6 +488,24 @@ export class ClipboardCellsState extends ClipboardCellsAbstractState {
     );
   }
 
+  private getHTMLContent(): string {
+    if (this.cells.length == 1 && this.cells[0].length == 1) {
+      return this.getters.getCellText(this.cells[0][0].position);
+    }
+
+    let htmlTable = '<table border="1" style="border-collapse:collapse">';
+    for (const row of this.cells) {
+      htmlTable += "<tr>";
+      for (const cell of row) {
+        const cssStyle = cssPropertiesToCss(cellTextStyleToCss(cell.style), false);
+        htmlTable += `<td style="${cssStyle}">` + this.getters.getCellText(cell.position) + "</td>";
```

Comment:
The cell content needs to be escaped in case it contains some html characters (the text `"<div/>"` in a cell should also be in the table)